### PR TITLE
Headers pages roy

### LINKED ISSUE #1: feat: add header and menu directories

### PR DESCRIPTION
**Pull request by padotada**
I want to merge.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<doctype html>
```
Comment (Bl20052005) [inline index.html]: do not change this :)

File: src/Components/MenuDirectories.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+nav > ul{
+    display:flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    align-items: center;
+    padding-inline-start: 0;
+    margin-block: 0;
+    height: 100%;
+}
+li:hover{
```
Comment (Bl20052005) [inline src/Components/MenuDirectories.module.css:15]: we probably want to use more specific names using className=... instead of just "li" or "ul"

**Review by Bl20052005 — CHANGES_REQUESTED** [review]

**Review by T-LXY — CHANGES_REQUESTED** [review]
Nice try

File: index.html (hunk 1 above)
Comment (padotada) [reply index.html]: okay

**Review by Bl20052005 — APPROVED** [review]
looks funny (good) I think

**Review by T-LXY — APPROVED** [review]
Better try

**Pull request merged**